Makefile

```diff
@@ -2,8 +2,23 @@
 # Setup Makefile to match your environment
 #
 PYTHON=/usr/bin/python3
-PYFLAKES=/usr/bin/pyflakes-3
-PYCODESTYLE=/usr/bin/pycodestyle-3
+
+# check for executables
+
+CMD := $(shell command -v pyflakes-3 2> /dev/null)
+ifndef CMD
+	PYFLAKES=/usr/bin/pyflakes
+else
+	PYFLAKES=/usr/bin/pyflakes-3
+endif
+
+CMD := $(shell command -v pycodestyle-3 2> /dev/null)
+ifndef CMD
+	PYCODESTYLE=/usr/bin/pycodestyle
+else
+	PYCODESTYLE=/usr/bin/pycodestyleî-3
+endif
+
```

Comment:
Be careful there is a special letter in pycodestyleî-3

I guess we do not need the Makefile anymore at all?

At least not the test/QA parts you touched.
In open suse build service we now have a separate cobbler-test package
@SchoolGuy Re-worked the whole test suite. He may have forgotten the one or other detail, like the README he adjusted recently:
tests/README.md

The Makefile parts are probably out of date and if, it would be better to revert them instead of having duplicate code/examples?

Reply:
I corrected this. If you are just using github, the Makefile is imho still needed. At least I'm building with "make rpms"

Reply:
Thanks for adding this to the Makefile. Although when using an IDE with CI this is technically not needed but since a lot of people are working in different ways this is a great addition. Before you merge please change the letter like @watologo1 stated.